frontend/yarn.lock

```diff
@@ -1032,62 +1032,62 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@contentful/experiences-core@npm:1.33.2":
-  version: 1.33.2
-  resolution: "@contentful/experiences-core@npm:1.33.2"
+"@contentful/experiences-core@npm:1.30.3":
+  version: 1.30.3
+  resolution: "@contentful/experiences-core@npm:1.30.3"
   dependencies:
-    "@contentful/experiences-validators": "npm:1.33.2"
+    "@contentful/experiences-validators": "npm:1.30.3"
     "@contentful/rich-text-types": "npm:^17.0.0"
   peerDependencies:
     contentful: ">=10.6.0"
-  checksum: 10c0/6a2668a175aa7d4ae72ac5baf481b5c0f0c586dbaf322c7b7067bfdd0db2071281599049dd19c320a84e95c5b2b0c52c3bd05745f6731dd93a49b963731379e6
+  checksum: 10c0/d45b784691d2bcf29540b03960ab2fe5221329d45d4accfd2c4d546390b69eaddcf7f50fceecf0186a1acd5c60cfded64a4654ebee37623508a323335dbda8d7
   languageName: node
   linkType: hard
 
-"@contentful/experiences-sdk-react@npm:^1.24.0":
-  version: 1.24.0
-  resolution: "@contentful/experiences-sdk-react@npm:1.24.0"
+"@contentful/experiences-sdk-react@npm:1.30.3":
+  version: 1.30.3
+  resolution: "@contentful/experiences-sdk-react@npm:1.30.3"
   dependencies:
-    "@contentful/experiences-components-react": "npm:1.24.0"
-    "@contentful/experiences-core": "npm:1.24.0"
-    "@contentful/experiences-validators": "npm:1.24.0"
-    "@contentful/experiences-visual-editor-react": "npm:1.24.0"
-    "@contentful/rich-text-types": "npm:^16.2.1"
+    "@contentful/experiences-components-react": "npm:1.30.3"
+    "@contentful/experiences-core": "npm:1.30.3"
+    "@contentful/experiences-validators": "npm:1.30.3"
+    "@contentful/experiences-visual-editor-react": "npm:1.30.3"
+    "@contentful/rich-text-types": "npm:^17.0.0"
     classnames: "npm:^2.3.2"
     csstype: "npm:^3.1.2"
     immer: "npm:^10.0.3"
     lodash-es: "npm:^4.17.21"
     md5: "npm:^2.3.0"
     style-inject: "npm:^0.3.0"
-    vite-plugin-css-injected-by-js: "npm:^3.1.1"
   peerDependencies:
     contentful: ">=10.6.0"
     react: ">=16.0.0"
     react-dom: ">=16.0.0"
-  checksum: 10c0/1d99da7fa3272f277e0430638cbd37fc536934070f51ae1d5e624625f4f938e1b0438f0db253f08df20d1566e8e32687b16138003beae14ae4d59a1017e5e190
+  checksum: 10c0/d07114d176f7236a3ee09a68e7954b1c14f0262f7f4274e601cb55adc9da3f49f35d8f5fb5c8fd181ee37b9416c0d1d2fe9219b3e113faeb38f17fa2678ee23e
   languageName: node
   linkType: hard
 
-"@contentful/experiences-sdk-react@npm:^1.33.2":
-  version: 1.33.2
-  resolution: "@contentful/experiences-sdk-react@npm:1.33.2"
+"@contentful/experiences-sdk-react@npm:^1.24.0":
```

Comment:
Do all of the `1.24.0` versions need to be `1.30.3` too?

Reply:
Thanks, I forgot to update the same in component library and pushed an update to the PR